docs/developer-docs/build/languages/motoko/actor-classes.md

```diff
@@ -1,18 +1,14 @@
 # Actor classes
 
-Actor classes enable you to create networks of actors (canisters) *programmatically*. Currently, actor classes have to be defined in a separate source file. To illustrate how to define and import actor classes, the following example implements a distributed map of keys of type `Nat` to values of type `Text`. It provides simple insert and lookup functions, `put(k, v)` and `get(k)`, for working with these keys and values.
+Actor classes enable you to create networks of actors (canister smart contracts) *programmatically*. Currently, actor classes have to be defined in a separate source file. To illustrate how to define and import actor classes, the following example implements a distributed map of keys of type `Nat` to values of type `Text`. It provides simple insert and lookup functions, `put(k, v)` and `get(k)`, for working with these keys and values.
 
 To distribute the data for this example, the set of keys is partitioned into `n` buckets. For now, we just fix `n = 8`. The bucket, `i`, of a key, `k`, is determined by the remainder of `k` divided by `n`, that is, `i = k % n`. The `i`th bucket (`i` in `[0..n)`) receives a dedicated actor to store text values assigned to keys in that bucket.
 
 The actor responsible for bucket `i` is obtained as an instance of the actor class `Bucket(i)`, defined in the sample `Buckets.mo` file, as follows:
 
-<div class="formalpara-title">
+`Buckets.mo`:
```

Comment:
It's really cool to see the interpreter working in conjunction with the adocs! Just curious, is this the expected layout output you want to see? for "Buckets.mo" text?
<img width="991" alt="screenshot" src="https://user-images.githubusercontent.com/38238542/174874224-fe12e894-957c-4c4d-a38b-d1240c30a104.png">